### PR TITLE
Bump wc dependency and add test for clear button

### PR DESCRIPTION
**Pull request by yuriy-fix**
Connected to https://github.com/vaadin/vaadin-combo-box-flow/issues/277

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-combo-box-flow/281)

### CODE REVIEW

File: vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/ClearValueIT.java [hunk 1]
```diff
@@ -41,6 +40,23 @@ public void valueIsCorrectlyCleared() {
...
+                "Unexpected selected item label for combo box with id '%s'",
+                comboBoxId), ClearValuePage.INITIAL_VALUE,
+                comboBox.getSelectedText());
+
+        comboBox.$("vaadin-text-field").get(0).$("[part~='clear-button']").get(0).click();
+        comboBox.openPopup();
+
+        Assert.assertEquals(String.format(
+                "Combo box with id '%s' should have its value empty after the test",
+                comboBoxId), "", comboBox.getSelectedText());
```
Comment (pekam) [inline vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/ClearValueIT.java]: This verifies only that the text-field is empty in client-side. It should verify the server-side state of `ComboBox.getValue()`. You need to add a value-change-listener or a button which sets some text on the page to e.g. "null" if the combo box's value is `null`

Comment (yuriy-fix) [reply vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/ClearValueIT.java]: Done

File: vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/ClearValueIT.java [hunk 2]
```diff
@@ -41,6 +41,30 @@ public void valueIsCorrectlyCleared() {
...
+        String comboBoxId = ClearValuePage.COMBO_BOX_WITH_CLEAR_BUTTON_ID;
+        ComboBoxElement comboBox = $(ComboBoxElement.class).id(comboBoxId);
+        Assert.assertEquals(String.format(
+                "Unexpected selected item label for combo box with id '%s'",
+                comboBoxId), ClearValuePage.INITIAL_VALUE,
+                comboBox.getSelectedText());
+
+        comboBox.$("vaadin-text-field").get(0)
+                .$("[part~='clear-button']").get(0).click();
+        comboBox.openPopup();
```
Comment (pekam) [inline vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/ClearValueIT.java]: This shouldn't be needed. Now it doesn't test the scenario which we want: clicking the overlay should set the value to null on server-side immediately without needing to open the overlay.

Comment (yuriy-fix) [reply vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/ClearValueIT.java]: Updated.

**Pull request merged**